integration-tests/tests/utils/config.py

```diff
@@ -0,0 +1,53 @@
+import shutil
+from dataclasses import dataclass
+from pathlib import Path
+
+
+@dataclass(frozen=True)
+class PackageConfig:
```

Comment:
Another high-level comment. 
`logs_source_dir` and `test_root_dir` are not really part of a package, so I think we are really mixing the use of package config vs TestConfig.

Can we separate this class into two, so we have a `TestConfig` class owning 1. a `PackageConfig`, and 2.  `logs_source_dir` and `test_root_dir`?

@kirkrodrigues let me know if you agree.